flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -560,12 +576,29 @@ bool vtol_follower_control_loiter(float dT, float *hold_pos, float *att_adj) {
 		cmd.Pitch
 	};
 
+	const float CMD_THRESHOLD = 0.2f;
+	const float CMD_ALT_THRESHOLD = 0.4f;
```

Comment:
That's a good point.  Probably still makes sense to use the navigation configured ascent/descent rates, I think, but using the althold configured deadbands makes sense.